tests/Unit/Libraries/Cms/Form/Rule/FilePathRuleTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * @package     Joomla.UnitTest
+ * @subpackage  HTML
+ *
+ * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Tests\Unit\Libraries\Cms\Form\Rule;
+
+use Joomla\CMS\Form\Rule\FilePathRule;
+use Joomla\Tests\Unit\UnitTestCase;
+
+/**
+ * Test class for FilePathRule.
+ *
+ * @package     Joomla.UnitTest
+ * @subpackage  Form
+ * @since       __DEPLOY_VERSION__
+ */
+class FilePathRuleTest extends UnitTestCase
+{
+	/**
+	 * Test data for the testRule method
+	 *
+	 * @return  array
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function dataTest(): array
+	{
+		$xml = new \SimpleXMLElement('<field
+			name="file_path"
+			type="text"
+			label="COM_MEDIA_FIELD_PATH_FILE_FOLDER_LABEL"
+			description="COM_MEDIA_FIELD_PATH_FILE_FOLDER_DESC"
+			size="50"
+			default="images"
+			validate="filePath"
+		/>');
+
+		// These all pass today,
+		// BUT, Joomla 3.9.26 SHOULD break this test, as a security fix is applied, thus proving the test valuable
+		return [
+			[true, $xml, ''],
+			[true, $xml, '.images'],
+			[true, $xml, './images'],
+			[true, $xml, '.\images'],
+			[false, $xml, '../images'],
+			[false, $xml, '.../images'],
+			[true, $xml, 'c:\images'],
+			[true, $xml, '\\images'], // Means \images
+			[true, $xml, 'ftp://images'],
+			[true, $xml, 'http://images'],
+			[true, $xml, '/media'],
+			[true, $xml, '/administrator'],
+			[true, $xml, '/4711images'],
+			[true, $xml, '4711images'],
```

Comment:
```suggestion
			[true, $xml, '/4711images'],
			[true, $xml, 'media'],
			[true, $xml, 'administrator'],
			[true, $xml, '4711images'],
```
See https://github.com/joomla/joomla-cms/pull/32718#issuecomment-801737187 for explanation.

Reply:
Update: No, this change should be made in J4 when merging up the security fix from staging after my PR #32718 has been merged. Sorry for confusion. All fine here.

Reply:
Yes that's correct - as per my comment in the PR Text above:

> NOTE: These tests are designed for, and pass, with 4.0-dev as it is today. When the security fixes for Joomla 3.9.26 are rolled into 4.0-dev eventually (by @wilsonge) then these tests SHOULD start to fail and will need true/false tweaking so that those test cases that are fixed by the security patches in Joomla 3.9.26 are now returning false...